modules/ossm-supported-configurations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * ossm-supported-platforms-configurations-3x.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-supported-configurations_{context}"]
+= Supported configurations
+
+The following configurations are supported for the current release of {SMProductName}.
+
+[id="ossm-supported-platforms_{context}"]
+== Supported platforms
+
+The {SMProductName} Operator supports multiple versions of the `Istio` resource. Version {MaistraVersion} {SMProductShortName} control planes are supported on the following platform versions:
```

Comment:
Since `Istio` resource is a CR, we can remove the word "resource" from the sentence. 

```suggestion
The {SMProductName} Operator supports multiple versions of `Istio`. Version {MaistraVersion} {SMProductShortName} control planes are supported on the following platform versions:
```